src/components/CheckboxDropdown.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import { Dispatch, SetStateAction, useState } from "react";
+import { twMerge } from "tailwind-merge";
+
+type CheckboxDropdownProps = {
+  options: {
+    id: string;
+    name: string;
+  }[];
+  className?: string;
+  htmlName: string;
+};
+
+export function CheckboxDropdown({
+  options,
+  className,
+  htmlName,
+}: CheckboxDropdownProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selected, setSelected] = useState(options[0].name);
```

Comment:
I think you copied that from RadioDropdown, but this will show you only the most recent select and ignore what you selected previously